api/src/controller/http/errors.ts

```diff
@@ -4,9 +4,15 @@ export class CustomError extends Error {
       | NotFoundErrorType
       | MethodNotAllowedErrorType
       | InternalErrorType,
-    public message: string
+    public messages: string | string[]
   ) {
-    super(message);
+    super(Array.isArray(messages) ? messages[0] : messages);
+
+    if (Array.isArray(messages)) {
+      this.messages = messages;
+    } else {
+      this.messages = [messages];
+    }
```

Comment:
Array 인지 판별하는 코드가 여러 곳에 분포하고 있어서 관리에 용이하지 않을 것 같습니다. 명확하게 파트를 나누면 좋을 것 같습니다.

```suggestion
    switch (true) {
    case Array.isArray(messages):
        super(messages[0]);
        this.messages = messages;
    default:
        super(messages);
        this.messages = [messages];
    }
```

Reply:
[구두 논의 내용]
- Variadic argument 을 사용해 인자의 개수와 상관없이 로직을 동일하게 처리할 수 있도록 수정.

수정은 https://github.com/MovieReviewComment/Mr.C/pull/47/commits/398f4c19a65b4d5b52a7ffcc515bfac16edf3daf 에서 진행했습니다.